api/types/status.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	cephTypes "github.com/canonical/microceph/microceph/api/types"
+	microTypes "github.com/canonical/microcluster/v2/rest/types"
+	ovnTypes "github.com/canonical/microovn/microovn/api/types"
+)
+
+// Status is a set of status information from a cluster member.
+type Status struct {
+	// Name represents the cluster name for the member.
+	Name string `json:"name" yaml:"name"`
+
+	// Address represnts the cluster address for the member.
+	Address string `json:"address" yaml:"address"`
+
+	// Clusters is a list of cluster members for each service installed on the member.
+	Clusters map[ServiceType][]microTypes.ClusterMember `json:"clusters" yaml:"clusters"`
+
+	// OSDs is a list of all OSDs local to the member.
+	OSDs cephTypes.Disks `json:"osds" yaml:"osds"`
+
+	// CephServices is a list of all ceph services running on this member.
+	CephServices cephTypes.Services `json:"ceph_services" yaml:"ceph_services"`
```

Comment:
From what I have seen the code generally talks about services when it comes to the availability of LXD, MicroCeph and MicroOVN. 

Should we pick another name or simply say `CephRoles`? This would maybe also make it easier to have another column in the table that can also show LXD's roles.

Reply:
Roles might get mixed up with dqlite so I used `units`. If you still think roles are better, I can switch it.